advanced-achievements-plugin/src/main/resources/config.yml

```diff
@@ -188,6 +188,12 @@ AllAchievementsReceivedRewards:
   IncreaseMaxOxygen: 30
   Experience: 100
 
+# Title shown on the root advancement.
+RootAdvancementTitle: "Advanced Achievements"
+
+# Background shown on the Advanced Achievements advancement tab.
+AdvancementsBackground: "minecraft:textures/items/book_enchanted.png"
```

Comment:
Could we explain how to get this value a little more? Maybe add the following sentence to the comment: "Must be a resource location to any image in a resource pack."